subworkflows/local/fastqc.nf

```diff
@@ -0,0 +1,45 @@
+//
+// Check input samplesheet and get read channels
+//
+
+//TODO --> add skip_fastqc to params
+
+include { FASTQC } from '../../modules/nf-core/modules/fastqc/main'
+
+workflow FASTQC_CHECK {
+  take:
+  ch_fastq
+
+  main:
+  ch_fastq
+      .map { ch -> [ ch[0], ch[1] ] }
+      .set { ch_fastq }
+
+  /*
+   * FastQ QC using FASTQC
+   */
+  fastqc_zip     = Channel.empty()
+  fastqc_html    = Channel.empty()
+  fastqc_multiqc = Channel.empty()
+  fastqc_version = Channel.empty()
```

Comment:
Is this necessary?

Reply:
no, some are obsolete, which came from old code.